src/wkwebview/mod.rs

```diff
@@ -787,11 +787,15 @@ impl InnerWebView {
             run_file_upload_panel as extern "C" fn(&Object, Sel, id, id, id, id),
           );
 
-          // Disable media dialogs
-          ctl.add_method(
-            sel!(webView:requestMediaCapturePermissionForOrigin:initiatedByFrame:type:decisionHandler:),
-            request_media_capture_permission as extern "C" fn(&Object, Sel, id, id, id, id, id),
-          );
+          // Only disable media dialogs on macOS < 14.0
+          // https://tauri.app/v1/references/webview-versions/
+          if !platform_webview_system_version()?.parse::<i32>().is_ok_and(|version| version > 18) {
```

Comment:
```suggestion
          let webview_system_version = platform_webview_system_version()?.parse::<i32>();
          if webview_system_version.is_err() || webview_system_version.unwrap() < 19 {
```

Could be more readable?